plenum/test/view_change/view_change_with_delays/test_view_change_with_advancing_node.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+import sys
+
+from plenum.server.node import Node
+from plenum.test.delayers import cDelay
+from plenum.test.helper import sdk_send_random_and_check, \
+    sdk_send_random_requests, sdk_get_replies
+from plenum.test.stasher import delay_rules
+from stp_core.loop.eventually import eventually
+
+
+@pytest.mark.skip(reason="INDY-1303")
+def test_delay_commits(txnPoolNodeSet, looper,
+                       sdk_pool_handle,
+                       sdk_wallet_client,
+                       tconf):
+    """
+    #3
+
+    Test case:
+
+    disable normal view change to make tests deterministic
+    delay commits for all nodes except node X
+    send request
+    check ordered transaction in node X
+    start view_change
+    check end of view change for all nodes
+    switch off commits' delay
+    get reply (means that request was ordered in all nodes)
+    repeat
+    Expected result with correct view change:
+    transactions should be ordered normally
+    Expected result with current view change:
+    node X can't finish second transaction
+    """
+    tconf.unsafe.add("disable_view_change")
+    sdk_send_random_and_check(looper, txnPoolNodeSet, sdk_pool_handle,
+                              sdk_wallet_client, 1)
+    nodes_stashers = [n.nodeIbStasher for n in txnPoolNodeSet
+                      if n != txnPoolNodeSet[-1]]
+    for _ in range(2):
+        do_view_change_with_delayed_commits_on_all_but_one(txnPoolNodeSet,
```

Comment:
We check that 2 random requests will execute successful and we will receive reply. Now this test has assertion error in the last ordered() because except_node can't order new transaction.